doc/code/qml_debugging.rst

```diff
@@ -1,8 +1,25 @@
+.. role:: html(raw)
+   :format: html
+
 qml.debugging
 =============
-.. currentmodule:: pennylane.debugging
 
-.. automodule:: pennylane.debugging
+This module contains functionality for debugging quantum programs on simulator devices.
+
+:html:`<div class="summary-table">`
```

Comment:
Curious why we couldn't use the built-in sphinx ones?

Reply:
The builtin one is a module level summary that links the functions in the table using the path through the module (i.e `qml.debugging.debug_expval()`). In order to use the top level links we have to customize it.